tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist = py36, py37, py38, lint
```

Comment:
tox _is_ pretty snazzy, especially for distributed packages – here we configure it to build the distribution against all supported version of python, and run the tests against each. it also runs a de-linter on the code.

it can run any command, and so we can put most anything here. though, I generally try not to conflate _e.g._ "build" tasks with the "test" tasks here.

(and, that's part of why I tend to have a `manage.py` module in most projects, for tying together and documenting all such tasks.)